### PR TITLE
Shannon/profile style

### PR DESCRIPTION
**Pull request by shannonchu**
Style user profile page
Render "your information" and "groups you lead" partials using javascript

### CODE REVIEW

File: app/assets/stylesheets/user.scss [hunk 1]
```diff
@@ -4,16 +4,44 @@
...
+  border-radius: 8px;
+  height: 48px;
+}
+
+.menu p {
+  color: $ashby-gray;
+}
+
+.actions {
+  height: 250px;
```
Comment (andylouisqin) [inline app/assets/stylesheets/user.scss]: This is a strange style; it's rendering as a giant white patch. Is this the result of some copy-pasta?

**Comment by hidesminimally** [comment]
Need to fix 'Groups you lead' portion, and make different separate class names for Create Group page

**Comment by hidesminimally** [comment]
Need to add data alert before allowing you to remove yourself from the group you lead:
- If they're the only leader: "Since you are the only leader of this group, leaving this group will delete this group. If you do not wish to delete this group, please appoint a group leader before trying to leave the group" 
- else if they're a leader among many: "You will be leaving your role as group leader if you leave this group. Are you sure you want to leave?"
- else: none 

This doesn't need to be styled

**Comment by andylouisqin** [comment]
LGTM

**Pull request merged**